andela/riders-ah-frontend
# #165413075 user can follow and unfollow each other

Description
=======
This feature is about allowing an authenticated user to follow an author on authors haven.

Type of change
=======
N/A

How has it been tested
=======
- pull this branch
- run ```npm install```
- run ```npm start``` to start our react app
-login into authorshaven
-choose an article written by another author
-follow the author
- unfollow the author
-navigate to your profile
- Click on either the following or followers header
- a page showing your followers and who you follow will appear
- there you can follow and unfollow authors.

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#165413075](https://www.pivotaltracker.com/story/show/165413075)
